backend/src/LibExecution/ProgramTypes.fs

```diff
@@ -231,31 +288,27 @@ module DB =
       cols : List<Col> }
 
 module UserType =
-  type RecordField = { name : string; typ : Option<DType>; nameID : id; typeID : id }
-  type Definition = Record of List<RecordField>
+  // TODO: move this to some ComplexType or CustomType type
```

Comment:
I find it's not super helpful to have incomplete thoughts added as comments, would prefer to remove them unless they're explanations of why the code is how to it (in which case longer explanations are more useful).